recipes/shop_manager_challenge_table_design.md

```diff
@@ -0,0 +1,158 @@
+# Two Tables Design Recipe Template
+
+_Copy this recipe template to design and create two related database tables from a specification._
+
+## 1. Extract nouns from the user stories or specification
+
+```
+# EXAMPLE USER STORY:
+# (analyse only the relevant part - here the final line).
+
+As a shop manager
+So I can know which items I have in stock
+I want to keep a list of my shop items with their name and unit price.
+
+As a shop manager
+So I can know which items I have in stock
+I want to know which quantity (a number) I have for each item.
+
+As a shop manager
+So I can manage items
+I want to be able to create a new item.
+
+As a shop manager
+So I can know which orders were made
+I want to keep a list of orders with their customer name.
+
+As a shop manager
+So I can know which orders were made
+I want to assign each order to their corresponding item.
+
+As a shop manager
+So I can know which orders were made
+I want to know on which date an order was placed. 
+
+As a shop manager
+So I can manage orders
+I want to be able to create a new order.
+```
+
+```
+Nouns:
+
+item, ,new_item, name, quantity_unit, unit_price
+order, new_order, item_name, order_date, customer_name 
+```
+
+## 2. Infer the Table Name and Columns
+
+Put the different nouns in this table. Replace the example with your own nouns.
+
+| Record                | Properties                                        |
+| --------------------- | --------------------------------------------------|
+| item                  |  name, unit_price, quantity
+| order                 |  item_name, order_date, customer_name 
+
+1. Name of the first table (always plural): `items` 
+
+    Column names: `name`, `quantity_unit`, `unit_price`
```

Comment:
What do you think about naming it just quantity and price? I also added unit_ to this, but I thought it might not be necessary as the name of the table should make it clear that we are talking about a item quantity or item price. Let me know what you think :)